src/main/java/com/microsoft/sqlserver/jdbc/SQLServerCallableStatement.java

```diff
@@ -1247,7 +1249,7 @@ public Ref getRef(String parameterName) throws SQLException {
      * @return the index
      */
     private int findColumn(String columnName) throws SQLServerException {
```

Comment:
columnName -> userProvidedColumnName to keep consistency with the other findColumn method

Reply:
nvm, its not a big deal